### ISSUE TITLE
LFO amplitude parameter offset modulation range

### ISSUE THREAD

**Issue by Razz21** [Tonejs/Tone.js#777]
Hello there, 
I am little confused about LFO's `amplitude` parameter. According to docs, in range `-10 to 10` with `amplitude` set to 0.5, LFO should modulate between -5 and 5, but [example](https://tonejs.github.io/docs/14.7.39/LFO) graph's wave is offset from center of this range. Is this a bug or desired behaviour?

**Comment by tambien** [comment]
The LFO in the example's range is set to 400 to 4000. So the center value is 2200 and the full amplitude modulation is ±1800. If you set the amplitude to 0.5 the LFO would still be centered around 2200, but would be ±(1800/2) = 900. Giving it a range of 1300 to 3100.

**Comment by Razz21** [comment]
Looking at these graphs I need to disagree it is working as you described. Wave is offset from the middle of range 400-4000.
```javascript
return Tone.Offline(() => {
	const lfo = new Tone.LFO({min:400, max:4000, amplitude:0.5}).start().toDestination();
}, 0.5, 1);
```
![newplot(2)](https://user-images.githubusercontent.com/32792205/97670655-ff886200-1a86-11eb-8bd5-8d72ae5f37d9.png)
```javascript
return Tone.Offline(() => {
	const lfo = new Tone.LFO({min:400, max:4000, amplitude:0}).start().toDestination();
}, 0.5, 1);
```
![newplot(1)](https://user-images.githubusercontent.com/32792205/97670699-1a5ad680-1a87-11eb-97c5-23404bd224c5.png)

**Comment by tambien** [comment]
yes! you're totally right. thanks for the images.

**Comment by Razz21** [comment]
Looks like this "LFO" works as expected:
```javascript
return Tone.Offline(() => {
	const min=400,max=4000;
	const osc = new Tone.Oscillator({frequency:"4n"}).start();
	const amplitude = new Tone.Gain(0.5);
	const mult = new Tone.Multiply((max-min)/2); // multiply wave [-1, 1] by relative distance to center point [1800] and create bipolar range around 0 in new scale [-1800, 1800];
	const add = new Tone.Add((max+min)/2).toDestination(); // add offset [2200] to fit original range;
	osc.chain(amplitude, mult, add);
}, 0.5, 1);
```
![newplot](https://user-images.githubusercontent.com/32792205/97727762-105dc580-1ad1-11eb-8684-59870a7d4137.png)

**Comment by tambien** [comment]
the latest fix `tone@14.7.67` should fix this. thanks for finding that!
